solutions/src/main/java/ru/sberbank/school/task02/Converter.java

```diff
@@ -0,0 +1,54 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import ru.sberbank.school.task02.exception.WrongSymbolException;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class Converter implements FxConversionService {
+
+    private ExternalQuotesService externalQuotesService;
+
+    public Converter(ExternalQuotesService externalQuotesService) {
+        this.externalQuotesService = externalQuotesService;
+    }
+
+
+    @Override
+    public BigDecimal convert( ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        if(operation == null || symbol == null || amount == null){
+            throw new NullPointerException("");
+        }
+
+        if(amount.equals(BigDecimal.ZERO))
```

Comment:
WhitespaceAround: &#39;if&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a706680861>*